### ISSUE TITLE
How about maintain a Wiki for MatConvNet?

### ISSUE THREAD

**Issue by luzhi** [vlfeat/matconvnet#541]
I would like to suggest that we (the MatConvNet users) can create the Wiki pages for the important questions (i.e., those topics has been asked for times), resources (like papers cite MatConvNet or models implemented/re-implemented by MatConvNet) and development plan of MatConvNet. The "Issues" are currently the primary resources for me if I have any problem about MatConvNet. But that contains too many information to allowing me to find out the one I need quickly (even if the issues are up-to-date).

This can makes everybody quickly find out the answers to their questions. How do you guys think? Thanks!

**Comment by lenck** [comment]
I think it's a great idea! :) Just checked and it seems that the GitHub wiki system would be ideal for it. So it would be really great if anyone would like to contribute!
Also, another way would be to send pull requests to the FAQ section on the webpage, but maybe the GitHub wiki would be a simpler way how to manage this section...

**Comment by luzhi** [comment]
@lenck Yes, the GitHub wiki is possibly the best place to have these knowledge.
